addon/components/dashboard-agenda.js

```diff
@@ -8,6 +8,7 @@ export default Component.extend({
   userEvents: service(),
 
   layout,
+  areEventsSelectable: true,
```

Comment:
Not sure about this constant. What's the purpose?

Reply:
this controls whether pre-work events are linked in the UI, or not. the default is `false`, so i need to pass this down in order to turn it on.

Reply:
@jrjohnson just verified that this flag is not tied to permissions, it's just a switch that has to be explicitly flipped on, on a per-component basis.